server/src/test/java/io/crate/integrationtests/RoleManagementIntegrationTest.java

```diff
@@ -82,18 +85,24 @@ public void testSysUsersTable() {
         assertUserIsCreated("arthur");
         executeAsSuperuser("CREATE USER ford WITH (password = 'foo')");
         assertUserIsCreated("ford");
-        executeAsSuperuser("SELECT name, password, superuser FROM sys.users WHERE superuser = FALSE ORDER BY name");
+        executeAsSuperuser("CREATE ROLE a_role");
+        assertRoleIsCreated("a_role");
+        executeAsSuperuser("GRANT a_role to arthur");
+        executeAsSuperuser("SELECT * FROM sys.users WHERE superuser = FALSE ORDER BY name");
         // Every created user is not a superuser
```

Comment:
Minor suggestion: we already test in docs scenario with grantor = crate. 
Can we test another scenario here, were grantor is a non-superuser?

Reply:
Done, thx!